app/renderer/components/tabs/tabPage.js

```diff
@@ -32,23 +32,6 @@ class TabPage extends React.Component {
     this.onMouseLeave = this.onMouseLeave.bind(this)
```

Comment:
maybe remove it all given other handlers are not bound to the constructor but curious if that affects reduxComponent